internal/store/hpa.go

```diff
@@ -76,6 +78,73 @@ var (
 				}
 			}),
 		},
+		{
+			Name: "kube_hpa_spec_target_metric",
+			Type: metric.Gauge,
+			Help: "The metric specifications used by this autoscaler when calculating the desired replica count.",
+			GenerateFunc: wrapHPAFunc(func(a *autoscaling.HorizontalPodAutoscaler) *metric.Family {
+				targets := make([]*metric.Metric, 0, len(a.Spec.Metrics))
+				for _, m := range a.Spec.Metrics {
+					var metricName string
+					var value, average int64
+					valueLabel := "value"
```

Comment:
We don’t need this variable. Let’s just use the string directly. I think we can miss out on some string optimization for readability’s sake.

Reply:
I update this variable in one of the cases (highlighted in another comment), I can avoid that and remove the variable

Reply:
Oh, I see. I was a bit confused by that.